src/deadline/client/cli/_groups/manifest_group.py

```diff
@@ -228,28 +234,98 @@ def manifest_download(
     """
     Downloads input manifest of previously submitted job.
     """
-    raise NotImplementedError("This CLI is being implemented.")
+    logger: ClickLogger = ClickLogger(is_json=json)
+    if not os.path.isdir(download_dir):
+        raise NonValidInputError(f"Specified destination directory {download_dir} does not exist. ")
+
+    # setup config
+    config = _apply_cli_options_to_config(required_options={"farm_id", "queue_id"}, **args)
+    queue_id: str = config_file.get_setting("defaults.queue_id", config=config)
+    farm_id: str = config_file.get_setting("defaults.farm_id", config=config)
+
+    boto3_session: boto3.Session = api.get_boto3_session(config=config)
+
+    output = _manifest_download(
+        download_dir=download_dir,
+        farm_id=farm_id,
+        queue_id=queue_id,
+        job_id=job_id,
+        step_id=step_id,
+        boto3_session=boto3_session,
+        logger=logger,
+    )
+    logger.json(dataclasses.asdict(output))
 
 
 @cli_manifest.command(
     name="upload",
-    help="BETA - Uploads a job attachment manifest file to a Content Addressable Storage's Manifest store. If calling via --s3-cas-path, it is recommended to use with --profile for a specific AWS profile with CAS S3 bucket access.",
+    help="BETA - Uploads a job attachment manifest file to a Content Addressable Storage's Manifest store. If calling via --s3-cas-path, it is recommended to use with --profile for a specific AWS profile with CAS S3 bucket access. Check exit code for success or failure.",
 )
 @click.argument("manifest_file")
 @click.option("--profile", help="The AWS profile to use.")
-@click.option("--s3-cas-path", help="The path to the Content Addressable Storage root.")
+@click.option("--s3-cas-uri", help="The URI to the Content Addressable Storage S3 bucket and root.")
```

Comment:
From reading the code below, this uri contains more than bucket name and root prefix?

Reply:
How so? I only use the bucket and prefix from it.

Reply:
From L320 and L330 below, seems the manifest path is also included?

Reply:
See below, it is just the bucket and root.